internal/logging/logging.go

```diff
@@ -28,5 +29,10 @@ func InitLogger() {
 }
 
 func GetLogger() *logrus.Logger {
```

Comment:
I just felt we can combine `initLogger()` and `GetLogger()`?

Reply:
`GetLogger()` will help us to get the global instance of `log` which will be initialized once in the whole program execution. 
I am implementing the singleton design pattern here. 

In future we will be initializing AWS cloudwatch logging under `initLogger()`

Reply:
then makes sense!